catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/test/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSourceTest.java

```diff
@@ -1724,11 +1724,7 @@ public void testSortingBadSortOrderWithoutDefault() throws Exception {
         ArgumentCaptor.forClass(ExtendedGetFeatureType.class);
 
     SourceResponse sourceResponse = source.query(new QueryRequestImpl(propertyIsLikeQuery));
-    assertThat(
-        sourceResponse
-            .getProperties()
-            .get(Wfs11Constants.UNSUPPORTED_SORT_BY_REMOVED + "." + WFS_ID),
-        is(Result.TEMPORAL));
+    assertThat(sourceResponse.getProcessingDetails().size(), is(1));
```

Comment:
✏️ Could use the `hasSize()` matcher instead, or even `contains()` if we want to verify the exact value.